recipe/conda_build_config.yaml

```diff
@@ -1,7 +1,9 @@
 rust_compiler_version:
-  - 1.64.0
+  - 1.76.0
 # orjson has wheels only for MacOS SDK 11.0, see https://pypi.org/project/orjson/3.8.8/#files
 MACOSX_DEPLOYMENT_TARGET:
-  - 11.0  # [osx and arm64]
+  - 10.12  # [osx and arm64]
 MACOSX_SDK_VERSION:
-  - "11.0"                 # [osx and arm64]
+  - "10.12"                 # [osx and arm64]
```

Comment:
The selectors don't match here-one's for arm and ones for x86. Is that what you intended?

Reply:
I fixed it